docs/getting_started.md

```diff
@@ -31,17 +34,57 @@ if (!("SingleCellExperiment" %in% installed.packages())) {
 
 library(SingleCellExperiment)
 # read in the RDS file, including the path to the file's location
-sce <- readRDS("SCPCS000000/SCPCL000000_filtered.rds")
+processed_sce <- readRDS("SCPCS000000/SCPCL000000_processed.rds")
 ```
 
 More resources for learning about `SingleCellExperiment` objects:
 
 - [`SingleCellExperiment` objects Vignette](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html) on working with `SingleCellExperiment` objects
 - [Orchestrating Single Cell Analysis chapter on the `SingleCellExperiment` class](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html)
 
-## Quality control
+## Working with the processed `SingleCellExperiment` objects
+
+The `processed` objects contain both the raw and normalized gene expression data, so many of the steps described in the section below can be skipped.
+These objects have already undergone quality control, removing low quality cells, normalization, and then dimensionality reduction like principal component analysis and UMAP.
+See the description of the {ref}`processed gene expression data <processing_information:Processed gene expression data>` for more information on the `processed` objects.
+
+The following command can be used to grab the normalized counts assay:
+
+```r
+# grab the log normalized data from the processed object
+normalized_counts <- logcounts(processed_sce)
+```
+
+These objects also contain a list of highly variable genes identified for the given library.
+These can be accessed using the following command:
+
+```r
+highly_variable_genes <- metadata(processed_sce)$highly_variable_genes
+```
```

Comment:
I might put this after the PCA, as the initial point of this list is to identify the genes that were used for PCA. If these are ranked (they are, correct?) then we should also mention that this list is ordered, as that would be useful information for use outside that context.